src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController.ts

```diff
@@ -216,6 +216,7 @@ export class TerminalChatController extends Disposable implements ITerminalContr
 			await this.reveal();
 		}
 		assertType(this._model.value);
+		this._messages.fire(Message.ACCEPT_INPUT);
```

Comment:
I don't really follow this one but I guess it was also related to some confusion in how terminal chat works vs other chats?